src/lib.rs

```diff
@@ -764,8 +768,12 @@ where
         // won't be dropped while the guard remains active.
         let n = unsafe { table.deref() }.bins.len();
 
-        // TODO: use num_cpus to help determine stride
-        let stride = MIN_TRANSFER_STRIDE;
+        lazy_static! {
```

Comment:
I'd rather not add a new dependency just for this. Can you modify this to use [`std::sync::Once`](https://doc.rust-lang.org/std/sync/struct.Once.html) instead? I think that should be sufficient.

Reply:
Sure